src/packagedcode/models.py

```diff
@@ -297,21 +334,21 @@ def compare_version(self, package, package_version=True):
 
         For example:
 
-        >>> q=Package(version='2')
-        >>> p=Package(version='1')
+        >>> q=Package(dict(versioning=Version(dict(version='2'))))
```

Comment:
This is the part we would not want: a direct depdency on the way schematics demands a dict as args.
We want instead to do:
`Package(versioning=Version(version='2'))`
or `Package(versioning=Version('2'))`